std/fs/constants.ts

```diff
@@ -0,0 +1,6 @@
+const { build } = Deno;
+
+import { EOL } from "./eol.ts";
+
+export const isWindows = build.os === "win";
+export const EOL_ = isWindows ? EOL.CRLF : EOL.LF;
```

Comment:
Why the underscore?

Reply:
Smallest difference? I couldn't think of a distinct name.

Reply:
Actually, I'll rename the enum to `Eol`. Pascal case is used for other enums.